app/components/Theme.ts

```diff
@@ -0,0 +1,11 @@
+export interface ThemeProps {
```

Comment:
Personally, id keep this one inline. by principle of least surprise, i would expect these to be colocated with the component that uses them.

as an aside, geekier note:
I think theres a tendency to have smaller files in js land to provide some amount of boundaries on what is possible which is less necessary in ts land.
this is an elm talk, but i think it applies to types in frontend code in general: https://www.youtube.com/watch?v=XpDsk374LDE. (not relevant for this pr, but watch it if your interested!)